index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
Here are few ideas for improving this project

1. Add a list to localStorage so it stays saved when the page is refreshed.
2. Don't allow adding empty tasks. Also, display a red error message below the input when the user tries to do so.
3. Add a checkbox in front of the task and cross it out when the user checks it.